pom.xml

```diff
@@ -10,8 +10,8 @@
 	<inceptionYear>2020</inceptionYear>
 
 	<properties>
-		<maven.compiler.target>1.9</maven.compiler.target>
-		<maven.compiler.source>1.9</maven.compiler.source>
+		<maven.compiler.target>1.8</maven.compiler.target>
```

Comment:
HI @mikera, This is a false pom.xml, my fork doesn't use maven but is under Gradle.

There is the pom file only why to this problem https://stackoverflow.com/questions/57903806/adding-used-by-inside-the-github-project-with-gradle-and-java

Reply:
I think you can simply re-create / overwrite these changes with any new update from the Gradle build.

Reply:
yes, we can like this code inside the build.gradle

```
allprojects {
    apply plugin: 'java'

    gradle.projectsEvaluated {
        tasks.withType(JavaCompile) {
            options.compilerArgs << "-Xlint:deprecation"
        }
    }
    sourceCompatibility = sourceCompatibility
    targetCompatibility = targetCompatibility
}
```

But with this message, I want to say only that the change inside pom.xml is unnecessary because the project don't use maven.

Reply:
OK, got it, thanks!